qualityassurancetopics.md

```diff
@@ -0,0 +1,75 @@
+## Main Endpoint
+Provide access to the Quality Assurance topics. A topic represents a specific type of event (such as a missing abstract can be).
+
+**GET /api/integration/qualityassurancetopics**
+It returns the list of the Quality Assurance Broker topics.
+
+```json
+[
+
+  {
+    id: "ENRICH!MORE!PID",
+    type: "qualityassurancetopic",
+    name: "ENRICH/MORE/PID",
+    lastEvent: "2020/10/09 10:11 UTC",
+    totalSuggestions: "33"
```

Comment:
I believe this field is now named `totalEvents`.  I don't see a `totalSuggestions` in the actual API response.